cmd/scaffold.go

```diff
@@ -808,3 +810,16 @@ func loadPrivateNodeInfo(dir string, myID flow.Identifier) (*bootstrap.NodeInfoP
 	err = json.Unmarshal(data, &info)
 	return &info, err
 }
+
+// IsValidNetworkAddressFormat ...
+func IsValidNetworkAddressFormat(str string) bool {
+	url, err := url.ParseRequestURI(str)
+	if err != nil {
+		return false
+	}
+	address := net.ParseIP(url.Host)
+	if address == nil {
+		return strings.Contains(url.Host, ".")
+	}
+	return true
+}
```

Comment:
This function considers pretty much every possible currently used value for this input to be invalid, which is why the integration tests and localnet are failing. For example, these values must be considered valid:
* `collection-001.mainnet10.nodes.onflow.org:9000`
* `localnet_access_1_1:9000`